cli/cmd/install.go

```diff
@@ -90,11 +93,13 @@ type (
 		highAvailability   bool
 		controllerUID      int64
 		disableH2Upgrade   bool
+		ignoreCluster      bool
```

Comment:
Since `ignoreCluster` is a configuration option for both inject and install, my preference is to add it to the (poorly named) `proxyConfigOptions` struct, which holds all of the other flag values that are shared between inject and install.